handlers/handlers.go

```diff
@@ -100,6 +100,30 @@ func HandleUserRequest(
 	}
 }
 
+func HandleRootRequest(
+	responseWriter http.ResponseWriter,
+	request *http.Request,
+) {
+	switch request.Method {
+	case http.MethodGet:
+		if request.URL.Path == "/" {
```

Comment:
why this? didn't we say we'd just have all "not founds" redirect to home for now? i could be remembering wrong though

Reply:
Your right we did talk about that, I'll do that, though i want to double check your okay with this "wonky" behavior. 

For example since we have `/user` handler,  `/user/banana` will result in HTTP.NotFound exception. However, `/bob/banana` will redirect home page since there is no `/bob` handler